src/dodal/beamlines/p38.py

```diff
@@ -1,12 +1,49 @@
-from dodal.devices.areadetector import AdAravisDetector
+from pathlib import Path
+
+from ophyd_async.core import DeviceCollector
+from ophyd_async.epics.areadetector import (
+    ADDriver,
+    HDFStreamerDet,
+    NDFileHDF,
+    TmpDirectoryProvider,
+)
```

Comment:
ophyd_async will look different in its next minor release; this can be merged for now but will need to be changed soon after anyways